src/crypto/equihash.tcc

```diff
@@ -12,18 +12,12 @@ bool DistinctIndices(const FullStepRow<WIDTH>& a, const FullStepRow<WIDTH>& b, s
 {
     std::vector<eh_index> aSrt = a.GetIndices(len, lenIndices);
     std::vector<eh_index> bSrt = b.GetIndices(len, lenIndices);
-
-    std::sort(aSrt.begin(), aSrt.end());
-    std::sort(bSrt.begin(), bSrt.end());
```

Comment:
Are these misnamed now (`Srt`) since they don't get sorted anymore? Or has `GetIndices` been changed in an earlier PR to return a sorted result?